code/scripting/api/libs/graphics.cpp

```diff
@@ -818,7 +840,7 @@ ADE_FUNC(drawModel, l_Graphics, "model model, vector position, orientation orien
 	matrix *orient = mh->GetMatrix();
 
 	//Clip
-	gr_set_clip(0, 0, gr_screen.max_w, gr_screen.max_h, GR_RESIZE_NONE);
+	gr_set_clip(0, 0, gr_screen.max_w, gr_screen.max_h, GR_RESIZE_NONE);	// don't use lua_ResizeMode here
```

Comment:
Optional: maybe explain why not? Ditto for the other spot where this happens.

Reply:
Done